.github/workflows/py-checkstyle.yml

```diff
@@ -17,6 +17,9 @@ on:
   pull_request_target:
     branches: [ main ]
 
+concurrency: 
+  group: ${{ github.head_ref }}
```

Comment:
let's update the group name to be `{workflow_name}-{github_head_ref}`.

Reply:
I have updated the group name as per the workflow name